### PR TITLE
test: add unit tests for PaymentInFlightExitModelUtils library

### PR DESCRIPTION
**Pull request by pgebal**
- Improves test coverage for `PaymentInFlightExitModelUtils` library to 100%.
- Moves `isFirstPiggybackOfTheToken` from `PaymentInFlightExitModelUtils` to `PaymentPiggybackInFlightExit` as it was used only there.
- Removes unused `clearOutputPiggyback` function.

References #389

### CODE REVIEW

File: plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+        it('fails for invalid output index', async () => {
+            await expectRevert(
+                this.modelUtils.clearOutputPiggybacked(MAX_OUTPUTS_NUM),
+                'Invalid output index',
+            );
+        });
+    });
+
+    describe('isInFirstPhase', () => {
```
[the remark this answers is not in the document]
Comment (pgebal) [reply plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js]: `MIN_EXIT_PERIOD / 2 - 1` is flaky, I put just the second case.

File: plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+const PaymentInFlightExitModelUtils = artifacts.require('PaymentInFlightExitModelUtilsMock');
+
+const { expectRevert, time } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+contract('PaymentInFlightExitModelUtils', () => {
+    const MAX_INPUTS_NUM = 4;
+    const MAX_OUTPUTS_NUM = 4;
+    const EXIT_MAP = 2 ** 1 | 2 ** 5; // piggyback input and output with index 1
```
[the remark this answers is not in the document]
Comment (pgebal) [reply plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js]: done

File: plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        it('returns true for piggybacked input', async () => {
+            this.modelUtils = await PaymentInFlightExitModelUtils.new(ALL_INPUTS_PIGGYBACKED, 0);
+
+            for (let i = 0; i++; i < MAX_INPUTS) {
+                const actual = await this.modelUtils.isInputPiggybacked(i);
+                expect(actual).to.be.true;
+            }
+        });
+
+        it('returns false for not piggybacked input', async () => {
```
Comment (boolafish) [inline plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js]:
```suggestion
        it('returns false for non-piggybacked input', async () => {
```

File: plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+            expect(actual).to.be.true;
+        });
+
+        it('returns false when ife is not in the first phase', async () => {
+            await time.increase(MIN_EXIT_PERIOD * 2);
+            const actual = await this.modelUtils.isInFirstPhase(MIN_EXIT_PERIOD);
+            expect(actual).to.be.false;
+        });
+
+        it('returns false when ife is not in the first phase - edge case', async () => {
```
Comment (boolafish) [inline plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js]:
```suggestion
        it('returns false when ife just passes the first phase (MIN_EXIT_PERIOD/2)', async () => {
```
